src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+
+import style from './Dashboard.module.scss';
+
+const activities = [
+  { activity: 'running', id: 1, timesDone: 3 },
+  { activity: 'swimming', id: 2, timesDone: 2 },
+  { activity: 'not smoking', id: 3, timesDone: 21 },
+];
+
+export const Dashboard = () => {
+  const [formVisible, setFormVisible] = useState<boolean>(false);
+  const [newActivity, setNewActivity] = useState<string>('');
+  const [currentActivities, setCurrentActivities] = useState([...activities]);
+
+  const handleChange = () => {
+    setNewActivity(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    event.preventDefault();
+    const newObj = { activity: newActivity, id: 4, timesDone: 0 };
```

Comment:
Since the ID is hardcoded, you can only add one item. But I take that as a temporary solution until the backend is in place.

So no worries